packages/trader/src/App/Components/Elements/VideoPlayer/volume-control.tsx

```diff
@@ -172,4 +172,4 @@ const VolumeControl = ({ onVolumeChange, volume, is_mobile, is_muted, toggleMute
     );
 };
 
-export default VolumeControl;
+export default React.memo(VolumeControl);
```

Comment:
re-renders:
- before: 4+ times per second during 1 video playing cycle;
- after: only when needed including: on the 1st video render, on the 1st controls appearance, and on every volume control actual re-render